sdk/cosmos/azure_data_cosmos/README.md

```diff
@@ -1,5 +1,200 @@
 # Azure Cosmos DB SDK for Rust.
 
-## Introduction
+This client library enables client applications to connect to Azure Cosmos DB via the NoSQL API. Azure Cosmos DB is a globally distributed, multi-model database service.
 
-This client library enables client applications to connect to [Azure Cosmos DB](https://azure.microsoft.com/en-us/products/cosmos-db) via the NoSQL API. Azure Cosmos DB is a globally distributed, multi-model database service.
+[Source code] | [Package (crates.io)] | [API reference documentation] | [Azure Cosmos DB for NoSQL documentation]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Cosmos DB SDK for Rust with cargo:
+
+```bash
+cargo add azure_data_cosmos
+```
+
+### Prerequisites
+
+* An [Azure subscription] or free Azure Cosmos DB trial account.
+
+Note: If you don't have an Azure subscription, create a free account before you begin.
+You can Try Azure Cosmos DB for free without an Azure subscription, free of charge and commitments, or create an Azure Cosmos DB free tier account, with the first 400 RU/s and 5 GB of storage for free. You can also use the Azure Cosmos DB Emulator with a URI of https://localhost:8081. For the key to use with the emulator, see [how to develop with the emulator](https://learn.microsoft.com/azure/cosmos-db/how-to-develop-emulator).
```

Comment:
`https` for localhost? Atypical, but doable if they accept to install the dev cert, which most aren't crazy about. Just wanted to ask since we all get in the habit of typing "https" these days.

Reply:
Alas, the Cosmos Emulator is only available over HTTPS, even on localhost. There's details in the linked doc on how to get the cert and install it.